lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -495,18 +504,20 @@ $(document).ready(function() {
     $('span.critic.comment').filter(function() {return $(this).text()!="";}).before('&#8225;');
   }
 
-  if($('#wiki-content').length ){
+  if($('.markdown-body').length ){
```

Comment:
Noticed that before, this `if` statement returned true on the `create` page.

Reply:
Ok.